src/basics/GlobalStyles.js

```diff
@@ -1,11 +1,32 @@
 import React from "react";
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle, css } from "styled-components";
 import styledNormalize from "styled-normalize";
 
 import { FONTS } from "constants/fonts";
-import { FONT_FAMILY } from "constants/styles";
+import { FONT_FAMILY, MEDIA_QUERIES } from "constants/styles";
 import { expansionStyles } from "components/Expansion";
 
+const linkedHeadingStyles = css`
```

Comment:
I think a comment explaining why this exception is needed (to the best of our knowledge anyway haha) would be good to have here

Reply:
Good point, PR comment is not enough. :+1: